delegation/src/Pos/Delegation/Cede/Types.hs

```diff
@@ -45,19 +47,25 @@ dlgEdgeActionIssuer = \case
     (DlgEdgeDel sId) -> sId
     (DlgEdgeAdd psk) -> addressHash (pskIssuerPk psk)
 
--- TODO should it use Pos.Util.Modifier ?
+-- Pos.Util.Modifier is something similar.
 -- | A set of modifications for dlg-related component to implement
 -- Cede-related typeclasses.
 data CedeModifier = CedeModifier
     { _cmPskMods            :: !(HashMap StakeholderId DlgEdgeAction)
       -- ^ PSK modifications (added/overwritten or deleted)
     , _cmHasPostedThisEpoch :: !(HashMap StakeholderId Bool)
       -- ^ 'True' means user has posted psk, 'False' means he didn't.
+    , _cmLookupCache        :: !(HashMap StakeholderId (Maybe ProxySKHeavy))
+      -- ^ This is a hacky cache to make things faster (it actually
+      -- does). It can also make things slower in the future. Cache
+      -- invalidation is hard, so we'll just `empty` it every time
```

Comment:
What do you mean by `empty` here? My interpretation was that whenever something is written, cache becomes `mempty`. But it's not the case: you only set `Nothing` for one key. So this sentence sounds confusing to me.

Reply:
I don't set `Nothing`, i set "nothing", this is how `at` works. I've updated the comment.